recipes/samarium/all/conandata.yml

```diff
@@ -2,3 +2,12 @@ sources:
   "1.0.0":
     sha256: 943de75803f492f78c5d55fe43be298fbb66156cc22946a3c6cc6b0634efc2e2
     url: https://github.com/strangeQuark1041/samarium/archive/refs/tags/v1.0.0.tar.gz
+  "1.0.1":
+    sha256: 3748ad1095805338d0d54ca7b60a155520188970946b247d56cc9b5eeb397358
+    url: https://github.com/strangeQuark1041/samarium/archive/refs/tags/v1.0.1.tar.gz
+patches:
+  "1.0.0":
+    - patch_file: "patches/1.0.0-updated-cmake.patch"
```

Comment:
It would be amazing if you could file in some of the metadata feilds so we are better able to track some ofthe patches https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches

as the author of the library you'd be prefect for filling in the blanks 🙏